src/widget/wlibrary.cpp

```diff
@@ -106,8 +106,8 @@ WTrackTableView* WLibrary::getCurrentTrackTableView() const {
     QWidget* pCurrent = currentWidget();
     WTrackTableView* pTracksView = qobject_cast<WTrackTableView*>(pCurrent);
     if (!pTracksView) {
-        // This view is no tracks view, but maybe a special tracks view with a
-        // controls row (DlgAutoDJ, DlgRecording)?
+        // This view is no tracks view, but maybe a special library view with a
+        // controls row and a track view (DlgAutoDJ, DlgRecording)?
```

Comment:
Were already cleaning up this comment, so it would make sense to fix all spelling & grammatical errors:
```
// This view is not a tracks view, but possibly a special library view
// with a controls row and a tracks view (DlgAutoDJ, DlgRecording,
// DlgHidden, DlgMissing, ...).
```

Reply:
Thanks!

FYI if you mark lines in the file diff tab you can click the +- button for change suggestions. This makes it easier to spot the difference.